### PR TITLE
Instrument permissions and name uniqueness

### PR DESCRIPTION
**Pull request by jasonrig**
* has_change_perm added, which checks the user's facility manager group membership.
* instrument name must now be unique

### CODE REVIEW

**Comment by jasonrig** [comment]
Note: This PR will conflict with PR #593 due to the migrations in each PR. PR #593 should be merged first, and then this simple migration can be recreated.

File: tardis/tardis_portal/models/instrument.py [hunk 1]
```diff
@@ -6,7 +6,7 @@ class Instrument(models.Model):
     '''
     Represents an instrument belonging to a facility that produces data
     '''
-    name = models.CharField(max_length=100)
+    name = models.CharField(max_length=100, unique=True)
     facility = models.ForeignKey(Facility)
 
     class Meta:
```
Comment (grischa) [inline tardis/tardis_portal/models/instrument.py]: Codacy Issue found: [Old-style class defined.](https://www.codacy.com/app/mytardis/mytardis/file/1888071676/issues/source?bid=2125231&fileBranchId=3089387#l12)

**Comment by jasonrig** [comment]
As discussed in person with @grischa: will update so the constraint is changed to enforce _name_ and _facility_ to be unique together. Changes will be committed as soon as PR #593 is addressed and my branch is rebased on develop.

**Comment by grischa** [comment]
Please also add a line to `docs/changes.rst`. Thanks

**Comment by grischa** [comment]
#593 has been merged, you can change it to unique_together facility and name now and make a new migration

**Pull request merged**